src/test/regress/expected/upgrade_type_after.out

```diff
@@ -0,0 +1,30 @@
+SET search_path TO upgrade_type, public;
+BEGIN;
+SET LOCAL citus.multi_shard_modify_mode TO 'sequential';
+\d tt
+              Table "upgrade_type.tt"
+ Column |  Type   | Collation | Nullable | Default 
+--------+---------+-----------+----------+---------
+ a      | integer |           | not null | 
+ b      | type1   |           |          | 
+Indexes:
+    "tt_pkey" PRIMARY KEY, btree (a)
+
+\dt type1
```

Comment:
Similarly, can you use something like `SELECT count(*) FROM pg_type WHERE typename ='type1' ` in order to guard us from future changes?

Reply:
It's not actually that important, since that would be testing pg_upgrade instead of the citus upgrade. So I removed it. The tests below still indirectly ensure that it exists.